### PR TITLE
Adding b2b `theme` to link component 

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: themes/theme-b2b/src/tokens.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const colors = {
     primary: {
       DEFAULT: brand.secondary[700],
     },
-
+    blue: '#3ab3d5',
```
Comment (sarahgm) [inline themes/theme-b2b/src/tokens.ts:35]: hm.. should we have a blue color? I'm wondering if we could have a better name for it. In core we used dark primary....

Comment (sarahgm) [reply themes/theme-b2b/src/tokens.ts:35]: maybe `link` ? or we make it a lighter gray and name it `primary-light`?

Comment (OsamaAbdellateef) [reply themes/theme-b2b/src/tokens.ts:35]: hmmm , for me i think `link` is better since this color isn't one of `brand` colors 
what do you think ?!

Comment (sarahgm) [reply themes/theme-b2b/src/tokens.ts:35]: yes. I would name it link

Comment (OsamaAbdellateef) [reply themes/theme-b2b/src/tokens.ts]: now switched to `link`

**Review by sarahgm — APPROVED** [review]

**Pull request merged**